### PR TITLE
Create navneetkarnam.md

### PR DESCRIPTION
**Pull request by navneetkarnam**

### CODE REVIEW

**Comment by aurinsomnia** [comment]
@navneetkarnam While this is a good start, try and add more markdown elements, get creative! Take inspiration from other pages as well if you need it. Also please edit your post and add either a screenshot or rawgit page, as that helps reviewers take a look at how the page looks on your end.

File: pages/profiles/navneetkarnam.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# NAVNEET KARNAM
+- Location: NEW PALTZ, NY
+- TimeZone: Eastern Standard Time
+- Operating System: Mac OS Sierra
+* * *
+## Summary: 
+> Strong information technology professional with a Master of Science (M.S.) focused in Computer Science from State University of New York at New Paltz. Experienced Information Technology Technician with a demonstrated history of working at the IT Help desk State University of New York at New Paltz. Skilled in Debugging Code, Java,JavaFX, HTML, CSS, AngularJs, MySQL and Arduino
+* * *
+
```
Comment (mappuji) [inline pages/profiles/navneetkarnam.md]: Why do you creating many blank lines? I think the markdown will not be rendered?

File: pages/profiles/navneetkarnam.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+
+
+
+
+
+## To know more about me:
+#### Linkedin: <https://www.linkedin.com/in/navneetkarnam>
```
Comment (mappuji) [inline pages/profiles/navneetkarnam.md]: Better to create a markdown link example:

`[title](http://)`

**Comment by kylemathias** [comment]
Looks like the video is down for me.
image other than that it looks good to me.

**Comment by navneetkarnam** [comment]
The video was private on youtube I just changed the privacy settings on it. It should be alright now.

**Pull request merged**